libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -252,14 +252,34 @@ export interface BucketDeleteOptions {
   readonly mustExist?: boolean;
 }
 
+/**
+ * Specifies the HTTP method permitted by a presigned URL for a bucket.
+ */
+export enum BucketSignedUrlAction {
+  /**
+   * Represents a GET request for a presigned URL, allowing read access for an object in the bucket.
+   */
+  GET = "GET",
+  /**
+   * Represents a PUT request for a presigned URL, allowing write access for an object in the bucket.
+   */
+  PUT = "PUT",
+}
+
 /**
  * Options for `Bucket.signedUrl()`.
  */
 export interface BucketSignedUrlOptions {
   /**
-   * The duration for the signed url to expire
+   * The duration for the signed URL to expire.
```

Comment:
what is the default?

Reply:
I didn't want to specify it there since it would be different based on the cloud provider:
- AWS is 15m.
- GCP is 1h.
- Azure forces you to specify it.

I think we can streamline it to be 15m for all of them.

Reply:
@Chriscbr FYI

Reply:
I feel like we should have the same defaults across targets unless there's an exceptional reason for one of them (e.g. related to `sim` being local). 15min or 1hr sounds alright as a default